lambda_functions/analyzer/analyzer_aws_lib.py

```diff
@@ -211,6 +211,14 @@ def _most_recent_item(self, sha: str) -> Optional[Tuple[int, Set[str], Set[str],
         else:
             return None
 
+    @staticmethod
+    def _replace_empty_strings(data: Dict[str, str]) -> Dict[str, str]:
+        """Replace empty string values, which are not allowed in Dynamo."""
+        for key, val in data.items():
+            if val == '':
```

Comment:
You may use `if not val:`.

Reply:
True, since in this case `data` is explicitly a string => string dictionary. But I prefer being explicit about the type here in case `data` becomes more general. For example, 0 is a valid dictionary key and shouldn't be replaced